apps/editing-toolkit/editing-toolkit-plugin/common/index.php

```diff
@@ -89,86 +66,6 @@ function needs_slider_width_workaround() {
 	return false;
```

Comment:
We can also remove `needs_slider_width_workaround` function as well as it's fixed by core and isn't required anymore.